Sheheroes_WatchOS/pubspec.lock

```diff
@@ -63,7 +63,7 @@ packages:
       name: async
       url: "https://pub.dartlang.org"
```

Comment:
remove this file from commit, lock file is not added usually.

Reply:
okay @hiashutoshsingh

Reply:
I have removed the padding field whose value is assigned to 0 . Can you guide me how to remove the file from commit ??

Reply:
@hiashutoshsingh  Can you please review the same ?